dynamic/help.1

```diff
@@ -76,5 +76,5 @@ After the agent tarball was unpacked, the (eventually mounted) files are being p
 
 .SH Documentation
 .PP
-Agent and general documentation is available at 
-\[la]https://docs.instana.com\[ra]
```

Comment:
docs.instana.com no longer works since the cert expired. Also, the URL has been outdated for a couple of years now.

Reply:
Thanks for updating